packages/core/src/external/surescripts/command/convert-batch-response/convert-batch-response-cloud.ts

```
@@ -0,0 +1,43 @@
+import { Config } from "../../../../util/config";
+import { getLambdaResultPayload, makeLambdaClient } from "../../../aws/lambda";
+import { SurescriptsConversionBundle, SurescriptsFileIdentifier } from "../../types";
+import { SurescriptsConvertBatchResponseHandler } from "./convert-batch-response";
+
+const region = Config.getAWSRegion();
+const lambdaClient = makeLambdaClient(region);
+
+export class SurescriptsConvertBatchResponseHandlerCloud
+  implements SurescriptsConvertBatchResponseHandler
+{
+  constructor(private readonly surescriptsConvertBatchResponseLambdaName: string) {}
+
+  async convertBatchResponse({
+    transmissionId,
+    populationId,
+  }: SurescriptsFileIdentifier): Promise<SurescriptsConversionBundle[]> {
+    const payload = JSON.stringify({ transmissionId, populationId });
+    try {
```

I'd remove this try catch and just let the error bubble up. You're not converting it into anything meaningful.